Tests/Approval.Tests/ApprovedApi/FluentAssertions/net47.verified.txt

```diff
@@ -1689,6 +1693,50 @@ namespace FluentAssertions.Numeric
         public FluentAssertions.AndConstraint<TAssertions> NotBeInRange(T minimumValue, T maximumValue, string because = "", params object[] becauseArgs) { }
         public FluentAssertions.AndConstraint<TAssertions> NotBeRankedEquallyTo(T unexpected, string because = "", params object[] becauseArgs) { }
     }
+    public class DecimalAssertions : FluentAssertions.Numeric.NumericAssertions<decimal>
+    {
+        public DecimalAssertions(decimal value) { }
+    }
+    public class DoubleAssertions : FluentAssertions.Numeric.NumericAssertions<double>
+    {
+        public DoubleAssertions(double value) { }
+    }
+    public class Int16Assertions : FluentAssertions.Numeric.NumericAssertions<short>
+    {
+        public Int16Assertions(short value) { }
+    }
+    public class Int32Assertions : FluentAssertions.Numeric.NumericAssertions<int>
+    {
+        public Int32Assertions(int value) { }
+    }
+    public class Int64Assertions : FluentAssertions.Numeric.NumericAssertions<long>
+    {
+        public Int64Assertions(long value) { }
+    }
+    public class NullableByteAssertions : FluentAssertions.Numeric.NullableNumericAssertions<byte>
+    {
+        public NullableByteAssertions(byte? value) { }
+    }
+    public class NullableDecimalAssertions : FluentAssertions.Numeric.NullableNumericAssertions<decimal>
+    {
+        public NullableDecimalAssertions(decimal? value) { }
+    }
+    public class NullableDoubleAssertions : FluentAssertions.Numeric.NullableNumericAssertions<double>
```

Comment:
FYI, that you had to add `NullableDoubleAssertions` led me to discover that we should have had it.
So I've opened #1867 to add it.